README.md

```diff
@@ -179,3 +179,14 @@ sbt publishLocal
 cd python
 pip install -e . # pip install -e .[all] to install all optional extras (see "Install from pypi")
 ```
+
+### Contribution
+#### Utilities
+`pre-commit` can be helpful in keep consistent code format with the repository. 
```

Comment:
=> "`pre-commit`[link here] is used to run various lints across the codebase to maintain consistent code style.`